src/aosm/azext_aosm/common/utils.py

```diff
@@ -130,5 +130,11 @@ def split_image_path(image) -> "tuple[str, str, str]":
     (name, version) = name_and_version.split(":", 2)
     return (source_acr_registry, name, version)
 
-def is_semver(string):
-    return re.match(SEMVER_REGEX, string) is not None
+def is_valid_nexus_image_version(string):
+    """Check if image version is valid.
+    
+    This is based on validation in pez repo.
+    It requires the image version to be major.minor.path,
+    but does enforce full semver validation.
```

Comment:
Not sure what line 138 means. Maybe it should be "but does not support full semver pre-release naming."?

Reply:
meant to be "does not" fixing